### PR TITLE
DOC: Fixup python-api docs slightly

### LINKED ISSUE #1502: Docs reference `mamba.mamba_api` but `libmambapy` should be used instead
Close this issue if this is incorrect but in the docs https://github.com/mamba-org/mamba/blob/129920a304084a59bdfbe026b036185a83d94e40/docs/source/python_api.rst users are told to use `from mamba import mamba_api`. For me within version `0.21.2` this import fails. Could this be related to https://github.com/mamba-org/mamba/pull/1219. I've found that using `import libmamba as mamba_api` works to the same effect.

### PR DESCRIPTION
**Pull request by HaoZeke**
Closes #1502.

### CODE REVIEW

File: docs/source/python_api.rst [hunk 1]
```diff
@@ -22,21 +22,21 @@ Using internal mamba APIs
 
 The core of ``mamba`` is written in C++, but we expose the internals of mamba with a Python API (using ``pybind11``).
 
-You can import the ``mamba_api`` using ``from mamba import mamba_api``.
+You can import ``libmambapy`` containing the Python API using ``from mamba.api import libmambapy``.
```
Comment (wolfv) [inline docs/source/python_api.rst:25]: I think `import libmambapy` works as well (not necessary to have `mamba` installed).

**Comment by wolfv** [comment]
thanks, this looks much better. Very much appreciated. 

I think only the import needs to be changed, then it's all good to me!

**Comment by jonashaag** [comment]
@HaoZeke

**Comment by HaoZeke** [comment]
Yup, this should be ready to go again :)

**Review by JohanMabille — APPROVED** [review]

**Comment by Nelly480** [comment]
CLEAR ALL ACCOUNTS

**Pull request merged**